src/main/java/com/amazon/opendistroforelasticsearch/security/ssl/rest/OpenDistroSecuritySSLReloadCertAction.java

```diff
@@ -0,0 +1,139 @@
+     *
+     * Sample response:
+     * { "message": "updated http certs" }
+     *
+     * @param request request to be served
+     * @param client client
+     * @throws IOException
+     */
+    @Override
+    protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) throws IOException {
+        return new RestChannelConsumer() {
+
+            final String certType = request.param("certType");
+
+            @Override
+            public void accept(RestChannel channel) throws Exception {
+                XContentBuilder builder = channel.newBuilder();
+                BytesRestResponse response = null;
+
+                // Check for Super admin user
+                final User user = (User) threadContext.getTransient(ConfigConstants.OPENDISTRO_SECURITY_USER);
+                if(user ==null||!adminDns.isAdmin(user)) {
+                    response = new BytesRestResponse(RestStatus.FORBIDDEN, "");
+                } else {
+                    try {
+                        builder.startObject();
+                        if (odsks != null) {
+                            switch (certType) {
+                                case "http":
+                                    odsks.initHttpSSLConfig();
+                                    builder.field("message", "updated http certs");
+                                    builder.endObject();
+                                    response = new BytesRestResponse(RestStatus.OK, builder);
+                                    break;
+                                case "transport":
+                                    odsks.initTransportSSLConfig();
+                                    builder.field("message", "updated transport certs");
+                                    builder.endObject();
+                                    response = new BytesRestResponse(RestStatus.OK, builder);
+                                    break;
+                                default:
+                                    builder.field("message", "invalid uri path, please use /_opendistro/_security/api/ssl/http/reload or " +
+                                        "/_opendistro/_security/api/ssl/transport/reload");
+                                    builder.endObject();
+                                    response = new BytesRestResponse(RestStatus.FORBIDDEN, builder);
+                                    break;
+                            }
+                        } else {
+                            builder.field("message", "keystore is not initialized");
+                            builder.endObject();
+                            response = new BytesRestResponse(RestStatus.INTERNAL_SERVER_ERROR, builder);
+                        }
+                    } catch (final Exception e1) {
+                        builder = channel.newBuilder();
+                        builder.startObject();
+                        builder.field("error", e1.toString());
```

Comment:
The following test cases return the exception if an Invalid DN is set and if the reload property is not there in the elasticsearch.yml config (https://github.com/opendistro-for-elasticsearch/security/pull/238/files#diff-714682d00b2af7282796d8efc9812f56R173-R198). I'll add one more in there in case the reloaded certificate has an invalid expiry date. 

Let me know if there is any specific test that comes to your mind.